src/academy/devdojo/maratonajava/javacore/streams/test/StreamCollectorsTest04.java

```diff
@@ -0,0 +1,58 @@
+package academy.devdojo.maratonajava.javacore.streams.test;
+
+import academy.devdojo.maratonajava.javacore.streams.domain.Category;
+import academy.devdojo.maratonajava.javacore.streams.domain.LightNovel;
+
+import java.util.*;
+import java.util.function.BinaryOperator;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.groupingBy;
+
+public class StreamCollectorsTest04 {
+
+    private static List<LightNovel> lightNovels = new ArrayList<>(List.of(
+            new LightNovel("Death Note", 3.99, Category.DRAMA),
+            new LightNovel("Dragon Ball", 2.59, Category.FANTASY),
+            new LightNovel("Demon Slayer", 5.89, Category.ROMANCE),
+            new LightNovel("Demon Slayer", 5.89, Category.ROMANCE),
+            new LightNovel("Pokemon", 6.50, Category.DRAMA),
+            new LightNovel("Fullmetal alchemist", 5.99, Category.FANTASY),
+            new LightNovel("Monogatari", 4.99, Category.DRAMA),
+            new LightNovel("Overlord", 8.99, Category.FANTASY)
+    ));
+
+    public static void main(String[] args) {
+
+        //COUNT BY CATEGORY
+        final Map<Category, Long> collect = lightNovels.stream().collect(groupingBy(LightNovel::getCategory, Collectors.counting()));
+        System.out.println(collect);
+
```

Comment:
What is the pattern of case used on your comments? UPPER, lower or Pascal Case?

Reply:
Pascal